test/acceptance/tests.py

```diff
@@ -1101,6 +1112,94 @@ def test_train_self_peer_staff(self, peer_grades_me):
                 [(u"YOUR SELF ASSESSMENT", u"Good"), (u"YOUR SELF ASSESSMENT", u"Excellent")],
             ])
 
+@ddt.ddt
+class FeedbackOnlyTest(OpenAssessmentTest, FullWorkflowMixin):
+    """
+    Test for a problem that only accepts feedback. Will make and verify peer, self, and staff assessments.
+    """
+    def setUp(self):
+        super(FeedbackOnlyTest, self).setUp("feedback_only", staff=True)
+        self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
+
+    def generate_feedback(self, assessment_type, feedback_type):
+        return "{}: {} feedback".format(assessment_type, feedback_type)
+
+    def assess_feedback(self, self_or_peer=""):
+        if self_or_peer != "self" and self_or_peer != "peer":
+            raise AssertionError("assert_feedback only works for self or peer assessments")
+        page = self.self_asmnt_page if self_or_peer == "self" else self.peer_asmnt_page
+        page.wait_for_page()
+        page.submit_assessment()
+
+    @retry()
+    @attr('acceptance')
+    def test_feedback_only(self):
+        # Make submission
+        user, pwd = self.do_submission()
+
+        # Make self assessment
+        self.self_asmnt_page.visit()
+        self.self_asmnt_page.wait_for_page()
+        self.self_asmnt_page.provide_criterion_feedback(self.generate_feedback("self", "criterion"))
+        self.self_asmnt_page.provide_overall_feedback(self.generate_feedback("self", "overall"))
+        self.self_asmnt_page.assess("self", [0])
+        self.self_asmnt_page.wait_for_complete()
+        self.assertTrue(self.self_asmnt_page.is_complete)
+
+        # Staff assess all available submissions
+        self.do_staff_assessment(
+            options_selected = [0],
+            feedback=lambda feedback_type: self.generate_feedback("staff", feedback_type)
+        )
+
+        # Verify feedback appears from all assessments in student-viewable grade report
+        self.refresh_page()
+        self.grade_page.wait_for_page()
+        for i, assessment_type in enumerate(["staff", "self"]):
+            # Criterion feedback first
+            expected = self.generate_feedback(assessment_type, "criterion")
+            actual = self.grade_page.feedback_entry(1, i)
+            self.assertEqual(actual, expected)
+            # Then overall
+            expected = self.generate_feedback(assessment_type, "overall")
+            actual = self.grade_page.feedback_entry("feedback", i)
+            self.assertEqual(actual, expected)
+
+        # Verify that all score-bearing items are hidden
```

Comment:
As discussed - I'll add 2 additional assertions above for the 1-point option from each of staff and self, then make the comment say something along the lines of "Since we've verified the 6 reported answers that should exist, ensure there are no more"